### DISCUSSION TITLE
ALVR can't find SteamVR: '/home/chair/.config/openvr/openvrpaths.vrpath does not exist'

### DISCUSSION THREAD

**Discussion by im-a-chair** [alvr-org/ALVR#2284] [Q&A]
im trying to get steamVR running through my quest 2.
ALVR lauches but when i click launch SteamVR it says: '/home/chair/.config/openvr/openvrpaths.vrpath does not exist'

OS: Ubuntu 22.04.4 LTS
CPU: Intel(R) Core(TM) i5-2400 CPU @ 3.10GHz
GPU: AMD Radeon RX 460 Graphics

Logs:
> 16:07:59.361837594 WARN Ignoring unknown button type: 
EVENT Session(SessionConfig { server_version: Version { major: 20, minor: 9, patch: 1 }, drivers_backup: Some(DriversBackup { alvr_path: "/home/chair/.local/share/ALVR-Launcher/installations/v20.9.1/alvr_streamer_linux/lib64/alvr", other_paths: [] }), openvr_config: OpenvrConfig { eye_resolution_width: 800, eye_resolution_height: 900, target_eye_resolution_width: 800, target_eye_resolution_height: 900, tracking_ref_only: false, enable_vive_tracker_proxy: false, aggressive_keyframe_resend: false, adapter_index: 0, codec: 0, h264_profile: 0, refresh_rate: 60, use_10bit_encoder: false, use_full_range_encoding: false, encoding_gamma: 0.0, enable_hdr: false, force_hdr_srgb_correction: false, clamp_hdr_extended_range: false, enable_pre_analysis: false, enable_vbaq: false, enable_hmqb: false, use_preproc: false, preproc_sigma: 0, preproc_tor: 0, amd_encoder_quality_preset: 0, rate_control_mode: 0, filler_data: false, entropy_coding: 0, force_sw_encoding: false, sw_thread_count: 0, controller_is_tracker: false, controllers_enabled: false, body_tracking_vive_enabled: false, body_tracking_has_legs: false, enable_foveated_encoding: false, foveation_center_size_x: 0.0, foveation_center_size_y: 0.0, foveation_center_shift_x: 0.0, foveation_center_shift_y: 0.0, foveation_edge_ratio_x: 0.0, foveation_edge_ratio_y: 0.0, enable_color_correction: false, brightness: 0.0, contrast: 0.0, saturation: 0.0, gamma: 0.0, sharpening: 0.0, linux_async_compute: false, linux_async_reprojection: false, nvenc_quality_preset: 0, nvenc_tuning_preset: 0, nvenc_multi_pass: 0, nvenc_adaptive_quantization_mode: 0, nvenc_low_delay_key_frame_scale: 0, nvenc_refresh_rate: 0, enable_intra_refresh: false, intra_refresh_period: 0, intra_refresh_count: 0, max_num_ref_frames: 0, gop_length: 0, p_frame_strategy: 0, nvenc_rate_control_mode: 0, rc_buffer_size: 0, rc_initial_delay: 0, rc_max_bitrate: 0, rc_average_bitrate: 0, nvenc_enable_weighted_prediction: false, capture_frame_dir: "/tmp", amd_bitrate_corruption_fix: false, _controller_profile: 0 }, client_connections: {}, session_settings: SettingsDefault { video: VideoConfigDefault { bitrate: BitrateConfigDefault { gui_collapsed: false, mode: BitrateModeDefault { ConstantMbps: 30, Adaptive: BitrateModeAdaptiveDefault { gui_collapsed: true, saturation_multiplier: 0.95, max_bitrate_mbps: SwitchDefault { enabled: false, content: 100 }, min_bitrate_mbps: SwitchDefault { enabled: false, content: 5 }, max_network_latency_ms: SwitchDefault { enabled: false, content: 8 }, encoder_latency_limiter: SwitchDefault { enabled: true, content: EncoderLatencyLimiterDefault { max_saturation_multiplier: 0.9 } }, decoder_latency_limiter: SwitchDefault { enabled: true, content: DecoderLatencyLimiterDefault { gui_collapsed: false, max_decoder_latency_ms: 30, latency_overstep_frames: 90, latency_overstep_multiplier: 0.99 } } }, variant: ConstantMbps }, adapt_to_framerate: SwitchDefault { enabled: true, content: BitrateAdaptiveFramerateConfigDefault { framerate_reset_threshold_multiplier: 2.0 } }, history_size: 256, image_corruption_fix: false }, preferred_codec: CodecTypeDefault { variant: H264 }, foveated_encoding: SwitchDefault { enabled: true, content: FoveatedEncodingConfigDefault { force_enable: false, center_size_x: 0.45, center_size_y: 0.4, center_shift_x: 0.4, center_shift_y: 0.1, edge_ratio_x: 4.0, edge_ratio_y: 5.0 } }, color_correction: SwitchDefault { enabled: true, content: ColorCorrectionConfigDefault { brightness: 0.0, contrast: 0.0, saturation: 0.5, gamma: 1.0, sharpening: 0.5 } }, max_buffering_frames: 2.0, buffering_history_weight: 0.9, optimize_game_render_latency: true, encoder_config: EncoderConfigDefault { gui_collapsed: true, rate_control_mode: RateControlModeDefault { variant: Cbr }, filler_data: false, h264_profile: H264ProfileDefault { variant: High }, entropy_coding: EntropyCodingDefault { variant: Cavlc }, use_10bit: false, use_full_range: true, encoding_gamma: 1.0, enable_hdr: false, force_hdr_srgb_correction: false, clamp_hdr_extended_range: false, nvenc: NvencConfigDefault { gui_collapsed: true, quality_preset: EncoderQualityPresetNvidiaDefault { variant: P1 }, tuning_preset: NvencTuningPresetDefault { variant: LowLatency }, multi_pass: NvencMultiPassDefault { variant: QuarterResolution }, adaptive_quantization_mode: NvencAdaptiveQuantizationModeDefault { variant: Spatial }, low_delay_key_frame_scale: -1, refresh_rate: -1, enable_intra_refresh: false, intra_refresh_period: -1, intra_refresh_count: -1, max_num_ref_frames: -1, gop_length: -1, p_frame_strategy: -1, rate_control_mode: -1, rc_buffer_size: -1, rc_initial_delay: -1, rc_max_bitrate: -1, rc_average_bitrate: -1, enable_weighted_prediction: false }, amf: AmfConfigDefault { gui_collapsed: true, quality_preset: EncoderQualityPresetAmdDefault { variant: Speed }, enable_vbaq: false, enable_hmqb: false, use_preproc: false, preproc_sigma: 4, preproc_tor: 7, enable_pre_analysis: false }, software: SoftwareEncodingConfigDefault { force_software_encoding: false, thread_count: 0 } }, force_software_decoder: false, mediacodec_extra_options: DictionaryDefault { gui_collapsed: true, key: "", value: MediacodecDataTypeDefault { Float: 0.0, Int32: 0, Int64: 0, String: "", variant: Int32 }, content: [("operating-rate", MediacodecDataTypeDefault { Float: 0.0, Int32: 2147483647, Int64: 0, String: "", variant: Int32 }), ("priority", MediacodecDataTypeDefault { Float: 0.0, Int32: 0, Int64: 0, String: "", variant: Int32 }), ("vendor.qti-ext-dec-low-latency.enable", MediacodecDataTypeDefault { Float: 0.0, Int32: 1, Int64: 0, String: "", variant: Int32 })] }, transcoding_view_resolution: FrameSizeDefault { Scale: 1.0, Absolute: FrameSizeAbsoluteDefault { width: 2144, height: OptionalDefault { set: false, content: 1072 } }, variant: Absolute }, emulated_headset_view_resolution: FrameSizeDefault { Scale: 1.0, Absolute: FrameSizeAbsoluteDefault { width: 2144, height: OptionalDefault { set: false, content: 1072 } }, variant: Absolute }, preferred_fps: 72.0, adapter_index: 0, clientside_foveation: SwitchDefault { enabled: false, content: ClientsideFoveationConfigDefault { mode: ClientsideFoveationModeDefault { Static: ClientsideFoveationModeStaticDefault { level: ClientsideFoveationLevelDefault { variant: High } }, Dynamic: ClientsideFoveationModeDynamicDefault { max_level: ClientsideFoveationLevelDefault { variant: High } }, variant: Dynamic }, vertical_offset_deg: 0.0 } } }, audio: AudioConfigDefault { game_audio: SwitchDefault { enabled: true, content: GameAudioConfigDefault { gui_collapsed: true, device: OptionalDefault { set: false, content: CustomAudioDeviceConfigDefault { NameSubstring: "", Index: 0, variant: NameSubstring } }, mute_when_streaming: true, buffering: AudioBufferingConfigDefault { gui_collapsed: true, average_buffering_ms: 50, batch_ms: 10 } } }, microphone: SwitchDefault { enabled: true, content: MicrophoneConfigDefault { gui_collapsed: true, devices: MicrophoneDevicesConfigDefault { Custom: MicrophoneDevicesConfigCustomDefault { sink: CustomAudioDeviceConfigDefault { NameSubstring: "", Index: 0, variant: NameSubstring }, source: CustomAudioDeviceConfigDefault { NameSubstring: "", Index: 0, variant: NameSubstring } }, variant: Automatic }, buffering: AudioBufferingConfigDefault { gui_collapsed: true, average_buffering_ms: 50, batch_ms: 10 } } } }, headset: HeadsetConfigDefault { position_recentering_mode: PositionRecenteringModeDefault { Local: PositionRecenteringModeLocalDefault { view_height: 1.5 }, variant: LocalFloor }, rotation_recentering_mode: RotationRecenteringModeDefault { variant: Yaw }, controllers: SwitchDefault { enabled: true, content: ControllersConfigDefault { gui_collapsed: false, tracked: true, enable_skeleton: true, gestures: SwitchDefault { enabled: true, content: HandGestureConfigDefault { gui_collapsed: true, only_touch: true, pinch_touch_distance: 0.0, pinch_trigger_distance: 0.25, curl_touch_distance: 2.0, curl_trigger_distance: 2.5, joystick_deadzone: 40.0, joystick_offset_horizontal: 0.0, joystick_offset_vertical: 0.0, joystick_range: 1.0, activation_delay: 50, deactivation_delay: 100, repeat_delay: 100 } }, steamvr_pipeline_frames: 3.0, haptics: SwitchDefault { enabled: true, content: HapticsConfigDefault { gui_collapsed: true, intensity_multiplier: 1.0, amplitude_curve: 1.0, min_duration_s: 0.01 } }, emulation_mode: ControllersEmulationModeDefault { Custom: ControllersEmulationModeCustomDefault { serial_number: "ALVR Controller", button_set: VectorDefault { gui_collapsed: false, element: "/user/hand/left/input/a/click", content: [] } }, variant: Quest2Touch }, extra_openvr_props: VectorDefault { gui_collapsed: true, element: OpenvrPropertyDefault { TrackingSystemName: "", ModelNumber: "", SerialNumber: "", RenderModelName: "", WillDriftInYaw: false, ManufacturerName: "", TrackingFirmwareVersion: "", HardwareRevisionString: "", AllWirelessDongleDescriptions: "", ConnectedWirelessDongle: "", DeviceIsWireless: false, DeviceIsCharging: false, DeviceBatteryPercentage: 0.0, FirmwareUpdateAvailable: false, FirmwareManualUpdate: false, FirmwareManualUpdateURL: "", HardwareRevisionUint64: 0, FirmwareVersion: 0, FPGAVersion: 0, VRCVersion: 0, RadioVersion: 0, DongleVersion: 0, BlockServerShutdown: false, CanUnifyCoordinateSystemWithHmd: false, ContainsProximitySensor: false, DeviceProvidesBatteryStatus: false, DeviceCanPowerOff: false, FirmwareProgrammingTarget: "", DeviceClass: 0, HasCamera: false, DriverVersion: "", FirmwareForceUpdateRequired: false, ViveSystemButtonFixRequired: false, ParentDriver: 0, ResourceRoot: "", RegisteredDeviceType: "", InputProfilePath: "", NeverTracked: false, NumCameras: 0, CameraFrameLayout: 0, CameraStreamFormat: 0, AdditionalDeviceSettingsPath: "", Identifiable: false, BootloaderVersion: 0, AdditionalSystemReportData: "", CompositeFirmwareVersion: "", FirmwareRemindUpdate: false, PeripheralApplicationVersion: 0, ManufacturerSerialNumber: "", ComputedSerialNumber: "", EstimatedDeviceFirstUseTime: 0, ReportsTimeSinceVSync: false, SecondsFromVsyncToPhotons: 0.0, DisplayFrequency: 0.0, UserIpdMeters: 0.0, CurrentUniverseId: 0, PreviousUniverseId: 0, DisplayFirmwareVersion: 0, IsOnDesktop: false, DisplayMCType: 0, DisplayMCOffset: 0.0, DisplayMCScale: 0.0, EdidVendorID: 0, DisplayMCImageLeft: "", DisplayMCImageRight: "", DisplayGCBlackClamp: 0.0, EdidProductID: 0, DisplayGCType: 0, DisplayGCOffset: 0.0, DisplayGCScale: 0.0, DisplayGCPrescale: 0.0, DisplayGCImage: "", LensCenterLeftU: 0.0, LensCenterLeftV: 0.0, LensCenterRightU: 0.0, LensCenterRightV: 0.0, UserHeadToEyeDepthMeters: 0.0, CameraFirmwareVersion: 0, CameraFirmwareDescription: "", DisplayFPGAVersion: 0, DisplayBootloaderVersion: 0, DisplayHardwareVersion: 0, AudioFirmwareVersion: 0, CameraCompatibilityMode: 0, ScreenshotHorizontalFieldOfViewDegrees: 0.0, ScreenshotVerticalFieldOfViewDegrees: 0.0, DisplaySuppressed: false, DisplayAllowNightMode: false, DisplayMCImageWidth: 0, DisplayMCImageHeight: 0, DisplayMCImageNumChannels: 0, SecondsFromPhotonsToVblank: 0.0, DriverDirectModeSendsVsyncEvents: false, DisplayDebugMode: false, GraphicsAdapterLuid: 0, DriverProvidedChaperonePath: "", ExpectedTrackingReferenceCount: 0, ExpectedControllerCount: 0, NamedIconPathControllerLeftDeviceOff: "", NamedIconPathControllerRightDeviceOff: "", NamedIconPathTrackingReferenceDeviceOff: "", DoNotApplyPrediction: false, DistortionMeshResolution: 0, DriverIsDrawingControllers: false, DriverRequestsApplicationPause: false, DriverRequestsReducedRendering: false, MinimumIpdStepMeters: 0.0, AudioBridgeFirmwareVersion: 0, ImageBridgeFirmwareVersion: 0, ImuFactoryGyroBias: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ImuFactoryGyroScale: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ImuFactoryAccelerometerBias: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ImuFactoryAccelerometerScale: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ConfigurationIncludesLighthouse20Features: false, AdditionalRadioFeatures: 0, ExpectedControllerType: "", HmdTrackingStyle: 0, DriverProvidedChaperoneVisibility: false, HmdColumnCorrectionSettingPrefix: "", CameraSupportsCompatibilityModes: false, SupportsRoomViewDepthProjection: false, DisplaySupportsMultipleFramerates: false, DisplayColorMultLeft: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, DisplayColorMultRight: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, DisplaySupportsRuntimeFramerateChange: false, DisplaySupportsAnalogGain: false, DisplayMinAnalogGain: 0.0, DisplayMaxAnalogGain: 0.0, DashboardScale: 0.0, IpdUIRangeMinMeters: 0.0, IpdUIRangeMaxMeters: 0.0, HmdSupportsHDCP14LegacyCompat: false, HmdSupportsMicMonitoring: false, DriverRequestedMuraCorrectionMode: 0, DriverRequestedMuraFeatherInnerLeft: 0, DriverRequestedMuraFeatherInnerRight: 0, DriverRequestedMuraFeatherInnerTop: 0, DriverRequestedMuraFeatherInnerBottom: 0, DriverRequestedMuraFeatherOuterLeft: 0, DriverRequestedMuraFeatherOuterRight: 0, DriverRequestedMuraFeatherOuterTop: 0, DriverRequestedMuraFeatherOuterBottom: 0, AudioDefaultPlaybackDeviceId: "", AudioDefaultRecordingDeviceId: "", AudioDefaultPlaybackDeviceVolume: 0.0, AudioSupportsDualSpeakerAndJackOutput: false, AttachedDeviceId: "", SupportedButtons: 0, Axis0Type: 0, Axis1Type: 0, Axis2Type: 0, Axis3Type: 0, Axis4Type: 0, ControllerRoleHint: 0, FieldOfViewLeftDegrees: 0.0, FieldOfViewRightDegrees: 0.0, FieldOfViewTopDegrees: 0.0, FieldOfViewBottomDegrees: 0.0, TrackingRangeMinimumMeters: 0.0, TrackingRangeMaximumMeters: 0.0, ModeLabel: "", CanWirelessIdentify: false, Nonce: 0, IconPathName: "", NamedIconPathDeviceOff: "", NamedIconPathDeviceSearching: "", NamedIconPathDeviceSearchingAlert: "", NamedIconPathDeviceReady: "", NamedIconPathDeviceReadyAlert: "", NamedIconPathDeviceNotReady: "", NamedIconPathDeviceStandby: "", NamedIconPathDeviceAlertLow: "", NamedIconPathDeviceStandbyAlert: "", OverrideContainer: 0, UserConfigPath: "", InstallPath: "", HasDisplayComponent: false, HasControllerComponent: false, HasCameraComponent: false, HasDriverDirectModeComponent: false, HasVirtualDisplayComponent: false, HasSpatialAnchorsSupport: false, ControllerType: "", ControllerHandSelectionPriority: 0, variant: TrackingSystemName }, content: [] }, linear_velocity_cutoff: 0.05, angular_velocity_cutoff: 10.0, left_controller_position_offset: ArrayDefault { gui_collapsed: true, content: [0.0, 0.0, -0.11] }, left_controller_rotation_offset: ArrayDefault { gui_collapsed: true, content: [-20.0, 0.0, 0.0] }, left_hand_tracking_position_offset: ArrayDefault { gui_collapsed: true, content: [0.04, -0.02, -0.13] }, left_hand_tracking_rotation_offset: ArrayDefault { gui_collapsed: true, content: [0.0, -45.0, -90.0] }, button_mappings: OptionalDefault { set: false, content: DictionaryDefault { gui_collapsed: false, key: "/user/hand/left/input/a/click", value: VectorDefault { gui_collapsed: false, element: ButtonBindingTargetDefault { destination: "/user/hand/left/input/a/click", mapping_type: ButtonMappingTypeDefault { HysteresisThreshold: HysteresisThresholdDefault { value: 0.5, deviation: 0.05 }, BinaryToScalar: BinaryToScalarStatesDefault { off: 0.0, on: 1.0 }, Remap: RangeDefault { min: 0.0, max: 1.0 }, variant: Passthrough }, binary_conditions: VectorDefault { gui_collapsed: true, element: "/user/hand/left/input/trigger/touch", content: [] } }, content: [] }, content: [] } }, button_mapping_config: AutomaticButtonMappingConfigDefault { gui_collapsed: true, click_threshold: HysteresisThresholdDefault { value: 0.5, deviation: 0.05 }, touch_threshold: HysteresisThresholdDefault { value: 0.1, deviation: 0.05 }, force_threshold: 0.8 } } }, emulation_mode: HeadsetEmulationModeDefault { Custom: HeadsetEmulationModeCustomDefault { serial_number: "Unknown" }, variant: Quest2 }, extra_openvr_props: VectorDefault { gui_collapsed: true, element: OpenvrPropertyDefault { TrackingSystemName: "", ModelNumber: "", SerialNumber: "", RenderModelName: "", WillDriftInYaw: false, ManufacturerName: "", TrackingFirmwareVersion: "", HardwareRevisionString: "", AllWirelessDongleDescriptions: "", ConnectedWirelessDongle: "", DeviceIsWireless: false, DeviceIsCharging: false, DeviceBatteryPercentage: 0.0, FirmwareUpdateAvailable: false, FirmwareManualUpdate: false, FirmwareManualUpdateURL: "", HardwareRevisionUint64: 0, FirmwareVersion: 0, FPGAVersion: 0, VRCVersion: 0, RadioVersion: 0, DongleVersion: 0, BlockServerShutdown: false, CanUnifyCoordinateSystemWithHmd: false, ContainsProximitySensor: false, DeviceProvidesBatteryStatus: false, DeviceCanPowerOff: false, FirmwareProgrammingTarget: "", DeviceClass: 0, HasCamera: false, DriverVersion: "", FirmwareForceUpdateRequired: false, ViveSystemButtonFixRequired: false, ParentDriver: 0, ResourceRoot: "", RegisteredDeviceType: "", InputProfilePath: "", NeverTracked: false, NumCameras: 0, CameraFrameLayout: 0, CameraStreamFormat: 0, AdditionalDeviceSettingsPath: "", Identifiable: false, BootloaderVersion: 0, AdditionalSystemReportData: "", CompositeFirmwareVersion: "", FirmwareRemindUpdate: false, PeripheralApplicationVersion: 0, ManufacturerSerialNumber: "", ComputedSerialNumber: "", EstimatedDeviceFirstUseTime: 0, ReportsTimeSinceVSync: false, SecondsFromVsyncToPhotons: 0.0, DisplayFrequency: 0.0, UserIpdMeters: 0.0, CurrentUniverseId: 0, PreviousUniverseId: 0, DisplayFirmwareVersion: 0, IsOnDesktop: false, DisplayMCType: 0, DisplayMCOffset: 0.0, DisplayMCScale: 0.0, EdidVendorID: 0, DisplayMCImageLeft: "", DisplayMCImageRight: "", DisplayGCBlackClamp: 0.0, EdidProductID: 0, DisplayGCType: 0, DisplayGCOffset: 0.0, DisplayGCScale: 0.0, DisplayGCPrescale: 0.0, DisplayGCImage: "", LensCenterLeftU: 0.0, LensCenterLeftV: 0.0, LensCenterRightU: 0.0, LensCenterRightV: 0.0, UserHeadToEyeDepthMeters: 0.0, CameraFirmwareVersion: 0, CameraFirmwareDescription: "", DisplayFPGAVersion: 0, DisplayBootloaderVersion: 0, DisplayHardwareVersion: 0, AudioFirmwareVersion: 0, CameraCompatibilityMode: 0, ScreenshotHorizontalFieldOfViewDegrees: 0.0, ScreenshotVerticalFieldOfViewDegrees: 0.0, DisplaySuppressed: false, DisplayAllowNightMode: false, DisplayMCImageWidth: 0, DisplayMCImageHeight: 0, DisplayMCImageNumChannels: 0, SecondsFromPhotonsToVblank: 0.0, DriverDirectModeSendsVsyncEvents: false, DisplayDebugMode: false, GraphicsAdapterLuid: 0, DriverProvidedChaperonePath: "", ExpectedTrackingReferenceCount: 0, ExpectedControllerCount: 0, NamedIconPathControllerLeftDeviceOff: "", NamedIconPathControllerRightDeviceOff: "", NamedIconPathTrackingReferenceDeviceOff: "", DoNotApplyPrediction: false, DistortionMeshResolution: 0, DriverIsDrawingControllers: false, DriverRequestsApplicationPause: false, DriverRequestsReducedRendering: false, MinimumIpdStepMeters: 0.0, AudioBridgeFirmwareVersion: 0, ImageBridgeFirmwareVersion: 0, ImuFactoryGyroBias: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ImuFactoryGyroScale: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ImuFactoryAccelerometerBias: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ImuFactoryAccelerometerScale: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, ConfigurationIncludesLighthouse20Features: false, AdditionalRadioFeatures: 0, ExpectedControllerType: "", HmdTrackingStyle: 0, DriverProvidedChaperoneVisibility: false, HmdColumnCorrectionSettingPrefix: "", CameraSupportsCompatibilityModes: false, SupportsRoomViewDepthProjection: false, DisplaySupportsMultipleFramerates: false, DisplayColorMultLeft: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, DisplayColorMultRight: ArrayDefault { gui_collapsed: false, content: [0.0, 0.0, 0.0] }, DisplaySupportsRuntimeFramerateChange: false, DisplaySupportsAnalogGain: false, DisplayMinAnalogGain: 0.0, DisplayMaxAnalogGain: 0.0, DashboardScale: 0.0, IpdUIRangeMinMeters: 0.0, IpdUIRangeMaxMeters: 0.0, HmdSupportsHDCP14LegacyCompat: false, HmdSupportsMicMonitoring: false, DriverRequestedMuraCorrectionMode: 0, DriverRequestedMuraFeatherInnerLeft: 0, DriverRequestedMuraFeatherInnerRight: 0, DriverRequestedMuraFeatherInnerTop: 0, DriverRequestedMuraFeatherInnerBottom: 0, DriverRequestedMuraFeatherOuterLeft: 0, DriverRequestedMuraFeatherOuterRight: 0, DriverRequestedMuraFeatherOuterTop: 0, DriverRequestedMuraFeatherOuterBottom: 0, AudioDefaultPlaybackDeviceId: "", AudioDefaultRecordingDeviceId: "", AudioDefaultPlaybackDeviceVolume: 0.0, AudioSupportsDualSpeakerAndJackOutput: false, AttachedDeviceId: "", SupportedButtons: 0, Axis0Type: 0, Axis1Type: 0, Axis2Type: 0, Axis3Type: 0, Axis4Type: 0, ControllerRoleHint: 0, FieldOfViewLeftDegrees: 0.0, FieldOfViewRightDegrees: 0.0, FieldOfViewTopDegrees: 0.0, FieldOfViewBottomDegrees: 0.0, TrackingRangeMinimumMeters: 0.0, TrackingRangeMaximumMeters: 0.0, ModeLabel: "", CanWirelessIdentify: false, Nonce: 0, IconPathName: "", NamedIconPathDeviceOff: "", NamedIconPathDeviceSearching: "", NamedIconPathDeviceSearchingAlert: "", NamedIconPathDeviceReady: "", NamedIconPathDeviceReadyAlert: "", NamedIconPathDeviceNotReady: "", NamedIconPathDeviceStandby: "", NamedIconPathDeviceAlertLow: "", NamedIconPathDeviceStandbyAlert: "", OverrideContainer: 0, UserConfigPath: "", InstallPath: "", HasDisplayComponent: false, HasControllerComponent: false, HasCameraComponent: false, HasDriverDirectModeComponent: false, HasVirtualDisplayComponent: false, HasSpatialAnchorsSupport: false, ControllerType: "", ControllerHandSelectionPriority: 0, variant: TrackingSystemName }, content: [] }, tracking_ref_only: false, enable_vive_tracker_proxy: false, face_tracking: SwitchDefault { enabled: false, content: FaceTrackingConfigDefault { gui_collapsed: true, sources: FaceTrackingSourcesConfigDefault { combined_eye_gaze: true, eye_tracking_fb: true, face_tracking_fb: true, eye_expressions_htc: true, lip_expressions_htc: true }, sink: FaceTrackingSinkConfigDefault { VrchatEyeOsc: FaceTrackingSinkConfigVrchatEyeOscDefault { port: 9000 }, variant: VrchatEyeOsc } } }, body_tracking: SwitchDefault { enabled: false, content: BodyTrackingConfigDefault { gui_collapsed: true, sources: BodyTrackingSourcesConfigDefault { body_tracking_full_body_meta: SwitchDefault { enabled: true, content: BodyTrackingFullBodyMETAConfigDefault { gui_collapsed: true, enable_full_body: true } } }, sink: BodyTrackingSinkConfigDefault { VrchatBodyOsc: BodyTrackingSinkConfigVrchatBodyOscDefault { port: 9000 }, variant: FakeViveTracker }, tracked: true } } }, connection: ConnectionConfigDefault { stream_protocol: SocketProtocolDefault { variant: Udp }, client_discovery: SwitchDefault { enabled: true, content: DiscoveryConfigDefault { auto_trust_clients: false } }, on_connect_script: "", on_disconnect_script: "", avoid_video_glitching: false, packet_size: 1400, stream_port: 9944, web_server_port: 8082, osc_local_port: 9942, server_send_buffer_bytes: SocketBufferSizeDefault { Custom: 100000, variant: Maximum }, server_recv_buffer_bytes: SocketBufferSizeDefault { Custom: 100000, variant: Maximum }, client_send_buffer_bytes: SocketBufferSizeDefault { Custom: 100000, variant: Maximum }, client_recv_buffer_bytes: SocketBufferSizeDefault { Custom: 100000, variant: Maximum }, max_queued_server_video_frames: 1024, statistics_history_size: 256, aggressive_keyframe_resend: false, dscp: OptionalDefault { set: false, content: DscpTosDefault { ClassSelector: 7, AssuredForwarding: DscpTosAssuredForwardingDefault { class: 4, drop_probability: DropProbabilityDefault { variant: Low } }, variant: ExpeditedForwarding } } }, extra: ExtraConfigDefault { steamvr_launcher: SteamvrLauncherDefault { driver_launch_action: DriverLaunchActionDefault { variant: UnregisterOtherDriversAtStartup }, open_close_steamvr_with_dashboard: false }, capture: CaptureConfigDefault { startup_video_recording: false, rolling_video_files: SwitchDefault { enabled: false, content: RollingVideoFilesConfigDefault { duration_s: 5 } }, capture_frame_dir: "" }, logging: LoggingConfigDefault { show_notification_tip: true, prefer_backtrace: false, notification_level: LogSeverityDefault { variant: Warning }, client_log_report_level: SwitchDefault { enabled: true, content: LogSeverityDefault { variant: Error } }, show_raw_events: SwitchDefault { enabled: false, content: RawEventsConfigDefault { hide_spammy_events: false } }, log_to_disk: false, log_tracking: false, log_button_presses: false, log_haptics: false }, patches: PatchesDefault { linux_async_compute: false, linux_async_reprojection: false }, open_setup_wizard: false } } })
16:08:06.000597954 INFO GPU Encoder vendor: Mesa Gallium driver 23.2.1-1ubuntu3.1~22.04.2 for AMD Radeon RX 460 Graphics (polaris11, LLVM 15.0.7, DRM 3.54, 6.5.0-44-generic)
16:08:06.000632954 INFO Couldn't find AV1 profile. You unlikely to have hardware encoding for it.
16:08:06.006234519 ERROR/home/chair/.config/openvr/openvrpaths.vrpath does not exist

**Comment by FilipoDeveloperMan** [comment]
same issue.
